mysql-wordpress-pd/README.md

```diff
@@ -31,10 +38,10 @@ your editor added one.
 
 ```shell
 tr --delete '\n' <password.txt >.strippedpassword.txt && mv .strippedpassword.txt password.txt
-kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/local-volumes.yaml
+kubectl create -f https://raw.githubusercontent.com/kubernetes/examples/master/mysql-wordpress-pd/local-volumes.yaml
```

Comment:
it is a bit "funny", none of these examples are consistent in the way they ask you to fetch the manifests.